doc/manual.rst

```diff
@@ -1095,6 +1097,7 @@ lowest and highest value of the type. For example:
   type
     Subrange = range[0..5]
     PositiveFloat = range[0.0..Inf]
+    PositiveInt = range[0..high(int)]
```

Comment:
Nit: positive means greater than 0, "greater than or equal to 0" would be non-negative.

Reply:
```suggestion
    Positive* = range[1..high(int)] # as defined in `system`
```